### PR TITLE
Convert TopBar and main menu to primereact

### PR DESCRIPTION
**Pull request by toddburnside**
It looks like what I demoed in the UI meeting.

### CODE REVIEW

File: common/src/main/webapp/sass/explore.scss [hunk 1]
```diff
@@ -1648,17 +1682,6 @@ table.explore-border-table {
   }
 }
 
-.p-component.p-button.pl-tiny {
```
Comment (cquiroz) [inline common/src/main/webapp/sass/explore.scss:1651]: I guess these are now on lucuma-ui

File: explore/src/main/scala/explore/TopBar.scala [hunk 2]
```diff
@@ -69,10 +67,11 @@ object TopBar:
       .useState(false) // isProgramsOpen
       .useStateView(IsAboutOpen(false))
       .useState(false) // just to force rerenders
-      .useEffectResultWithDepsBy((_, _, _, _, toggle) => toggle.value)((_, _, _, _, _) =>
+      .usePopupMenuRef
```
Comment (cquiroz) [inline explore/src/main/scala/explore/TopBar.scala:70]: I guess this comes from lucuma-ui too?

**Review by cquiroz — APPROVED** [review]
LGTM

File: common/src/main/webapp/sass/explore.scss (hunk 1 above)
Comment (toddburnside) [reply common/src/main/webapp/sass/explore.scss:1651]: Yes. The lucuma-ui classes work for this now.

File: explore/src/main/scala/explore/TopBar.scala (hunk 2 above)
Comment (toddburnside) [reply explore/src/main/scala/explore/TopBar.scala:70]: That comes from lucuma-react

Comment (toddburnside) [reply explore/src/main/scala/explore/TopBar.scala:70]: It's like the ToastRef

**Pull request merged**